src/api/get-installations.js

```diff
@@ -41,7 +60,11 @@ async function findWindowsBrowsers () {
     var browserRe     = new RegExp(regKeyEsc + '([^\\\\]+)\\\\shell\\\\open\\\\command' +
                                    '\\s+\\([^)]+\\)\\s+reg_sz\\s+([^\n]+)\n', 'gi');
 
-    var stdout = await exec(`chcp 65001 | reg query ${regKey} /s`);
+    // NOTE: we should change code page to get correct result regardless the WIndows localization.
```

Comment:
@georgiy-abbasov Correct formatting a little:
```js
// NOTE: we need to change the code page to get the correct result regardless of
// the Windows localization. Then we need to restore it to avoid terminal errors.
```